brands/amenity/bank.json

```
@@ -2301,6 +2301,16 @@
       "name": "Banrural"
     }
   },
+  "amenity/bank|Bantierra": {
+    "countryCodes": ["es"],
+    "tags": {
+      "amenity": "bank",
+      "brand": "Bantierra",
+      "brand: wikipedia": "es: Bantierra",
```

Please remove the extra spaces:

```suggestion
      "brand:wikipedia": "es:Bantierra",
```

---

oh sorry, did not notice that.Thanks!